erpnext/projects/doctype/project/project.json

```diff
@@ -40,7 +40,9 @@
   "actual_start_date",
   "actual_time",
   "column_break_20",
+  "estimated_time",
```

Comment:
the `estimated_time` is twice in .json file.

Reply:
removed the repeated field from.json file